src/main/java/com/basho/riak/client/core/query/timeseries/CollectionConverters.java

```diff
@@ -104,16 +110,44 @@ private static FullColumnDescription convertDescribeResultRowToFullColumnDescrip
         final ColumnDescription.ColumnType type =
                 ColumnDescription.ColumnType.valueOf(typeString.toUpperCase(Locale.ENGLISH));
 
+        final Quantum quantum = parseQuantumCells(cells);
+
         return new FullColumnDescription(name,
                                          type,
                                          isNullable,
                                          partitionKeyOrdinal,
-                                         localKeyOrdinal);
+                                         localKeyOrdinal,
+                                         quantum);
+    }
+
+    private static Quantum parseQuantumCells(List<Cell> cells)
+    {
+        final boolean hasQuantum = cells.size() > 5 && cells.get(5) != null && cells.get(6) != null;
```

Comment:
Why don't us extract this line as a function and reuse it in DescribeFnRowResultIsValid() to replace the following condition:

```
final boolean isValidV2Description =
                cells.size() == 7 &&
                describeRowBaseIsValid(cells) &&
                cells.get(5) != null ? cells.get(5).hasLong() : true &&
                cells.get(6) != null ? cells.get(6).hasVarcharValue() : true;
```